packages/terra-framework-docs/CHANGELOG.md

```diff
@@ -3,26 +3,25 @@
 ## Unreleased
 
 * Added
-  * Added `aria-label` and `ariaDescribedBy` props.
+  * Added `aria-label` and `ariaDescribedBy` examples for `terra-collapsible-menu-view`.
+  * Added examples and test to `terra-table` for row selection mode.
+  * New Terra Slider component documentation, examples and tests.
```

Comment:
Having these under unreleased seems incorrect. I am sure Slider changes were released under 1.43.0 but might have to double check on the terra-table row selection mode changes

Reply:
You're right, corrected this in: 1862c0d65c7dc21b176895d911d397e042f420d9